pyrevitlib/rpw/db/builtins.py

```diff
@@ -155,8 +156,10 @@ def from_category_id(self, category_id):
         Returns:
             ``DB.BuiltInCategory`` member
         """
-        bic = Enum.ToObject(DB.BuiltInCategory, category_id.IntegerValue)
-        if DB.ElementId(bic).IntegerValue < -1:
+        value_func = get_value_func()
+        bic = Enum.ToObject(DB.BuiltInCategory, value_func(category_id))
+        bic_value = value_func(DB.ElementId(bic))
```

Comment:
To avoid back and forth between ElementIds, integers and enums, I would do like this (untested)

```python
cat_id_value = value_func(category_id)
if cat_id_value < -1 and Enum.IsDefined(DB.BuiltInCategory, cat_id_value):
    return Enum.ToObject(DB.BuiltInCategory, cat_id_value)
```